README.md

```diff
@@ -32,6 +32,23 @@ use a major version when expressing a dependency on `aws-sdk`.
 gem 'aws-sdk', '~> 2'
 ```
 
+##Getting Help
+
+Please use these community resources for getting help. We use the GitHub issues for tracking bugs and feature requests and have limited bandwidth to address them.
+
+* Ask a question on StackOverflow and tag it with `aws-sdk-ruby`
+* Come join the AWS Ruby community chat on gitter: `aws/aws-sdk-ruby`
+* Articulate your feature request or upvote existing ones
+* Open a support ticket with [AWS Support](https://console.aws.amazon.com/support/home), if it turns out that you may have found a bug, please open an issue
+
```

Comment:
I'd potentially add a note along the lines of:

* If in doubt as to whether your issue is a question about how to use AWS or a potential SDK issue, feel free to open a GitHub issue on this repo.